macros/upload_results.sql

```diff
@@ -126,14 +126,22 @@
 
         {% do log("Uploading sources", true) %}
         {% set sources = dbt_artifacts.get_relation('sources') %}
-        {% set content_sources = dbt_artifacts.upload_sources(graph) %}
-        {{ dbt_artifacts.insert_into_metadata_table(
-            database_name=sources.database,
-            schema_name=sources.schema,
-            table_name=sources.identifier,
-            content=content_sources
-            )
-        }}
+        {% set sources_set = [] %}
+        {% for node in graph.nodes.values() | selectattr("resource_type", "equalto", "source") %}
```

Comment:
```suggestion
        {% for node in graph.sources.values() %}
```

If we look in the `upload_sources` macro that was changed we can see it uses `graph.sources` rather than `graph.nodes`. 
 
Currently when this runs nothing is inserted into the sources tables.